simapp/mint_fn.go

```diff
@@ -0,0 +1,123 @@
+package simapp
```

Comment:
Because this is the example that developers can use to develop their own mint function. The idea here is that anyone creates their own minting function if they want to, and it works without requiring direct usage of keepers as it uses the query routers.

Reply:
Thanks for the explanation! Personally, I would prefer something like `x/mint/example`. I do understand your point of keeping it decoupled but it still relates more the mint package than to simapp for me. This is personal preference of course.